### PR TITLE
[nucleus-3.0] deprecate components

### PR DESCRIPTION
**Pull request by MekalaNagarajan-Centrica**

### CODE REVIEW

File: docs/components/ns-article.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { StorybookStory } from '../../includes/storybook-story.js'
 import { ComponentPlacement } from '../../includes/component-placement.js'
 import { PageFooter } from '../../includes/page-footer.js'
 
+## Deprecated
+> This component is deprecated.
```
Comment (jholt1) [inline docs/components/ns-article.md:10]:
```suggestion
> This component is currently being considered for deprecation.
```

File: docs/components/ns-content.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ import { StorybookStory } from '../../includes/storybook-story.js'
 import { ComponentPlacement } from '../../includes/component-placement.js'
 import { PageFooter } from '../../includes/page-footer.js'
 
+## Deprecated
+> This component is deprecated.
```
Comment (jholt1) [inline docs/components/ns-content.md:10]:
```suggestion
> This component is currently being considered for deprecation.
```

File: docs/components/ns-editorial.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ import { StorybookStory } from '../../includes/storybook-story.js'
 import { ComponentPlacement } from '../../includes/component-placement.js'
 import { PageFooter } from '../../includes/page-footer.js'
 
+## Deprecated
+> This component is deprecated.
```
Comment (jholt1) [inline docs/components/ns-editorial.md:10]:
```suggestion
> This component is currently being considered for deprecation.
```

File: docs/components/ns-article.md (hunk 1 above)
Comment (MekalaNagarajan-Centrica) [reply docs/components/ns-article.md]: done

File: docs/components/ns-content.md (hunk 2 above)
Comment (MekalaNagarajan-Centrica) [reply docs/components/ns-content.md]: done

**Comment by andij** [comment]
Could we bit a bit more subtle with the messaging.

This:

```markdown
## Introduction

> `ns-article` is used to display long-form content
>
> **Note:** This component is currently being considered for deprecation.
```

Instead of this:

```markdown
## Deprecated

> This component is currently being considered for deprecation.

## Introduction

> `ns-article` is used to display long-form content
```

**Review by andij — APPROVED** [review]

**Pull request merged**